src/DeezerAPI.php

```diff
@@ -111,6 +111,21 @@ public function getPlaylist($playlistId)
     }
 
     /**
+     * Get all playlists for the authenticated user
+     * 
+     * @return array
+     */
+    public function getPlaylists()
```

Comment:
Nipick: Since we got a `createUserPlaylist` method, maybe name this `getUserPlaylists` ?